xm-commons-cache/src/main/java/com/icthh/xm/commons/cache/config/XmTenantLepCacheConfig.java

```diff
@@ -0,0 +1,68 @@
+package com.icthh.xm.commons.cache.config;
+
+import com.github.benmanes.caffeine.cache.Ticker;
+import com.icthh.xm.commons.cache.service.DynamicCaffeineCacheManager;
+import com.icthh.xm.commons.cache.service.TenantAwareCacheManager;
+import com.icthh.xm.commons.cache.TenantCacheManager;
+import com.icthh.xm.commons.cache.service.XmCacheConfigurer;
+import com.icthh.xm.commons.tenant.TenantContextHolder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnProperty(value = "application.tenant-memory-cache.enabled", havingValue = "true")
```

Comment:
why do we need two distinct options? `application.tenant-cache.enabled` and `application.tenant-memory-cache.enabled` ?